### DISCUSSION TITLE
Errors using pac pcf push

### DISCUSSION THREAD

**Discussion by Yungkingdre** [microsoft/powerplatform-build-tools#545] [General]
I am getting this **error(pac pcf push --publisher-prefix contoso)** every time I use **pac pcf push --publisher-prefix contoso**.
Kindly help

**Comment by ghost** [comment]
I'm having the same issue on Mac Silicon. Seems to work on Windows

**Comment by petrochuk** [comment]
Can you post exact error and exception id?

**Comment by imastur** [comment]
I am having the same problem: This is the error: 
![image](https://github.com/microsoft/powerplatform-build-tools/assets/101188156/69f96189-9ec9-4355-a62e-e22d9a70db57)

**Reply by pavansarma93** [reply]
@imastur did you see what does the build log have to tell ? you can upload here
